instagram-page-react/src/pages/profilePage/molecules/profileTabbar/atoms/profileTabbarTab/ProfileTabbarTab.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+// Styles
+import styles from './profileTabbarTab.module.css'
+
+function ProfileTabbarTab(props) {
+    const {imageSrc,
+        Text,
+        isSelected
+    } = props;
+
+    const groupTabbarTabClassName = [
+        styles.profileTabbarTab,
+        isSelected ? styles.profileTabbarTabSelected : ''
+    ].join(' ')
+
+    const groupTabbarTabTextClassname = [
+        styles.profileTabbarTabText,
+        isSelected ? styles.profileTabbarTabTextSelected : ''
+    ].join(' ')
+
+    return (
+        <div className={groupTabbarTabClassName}>
+            <img alt="profile-tabbar-tab-icon"
```

Comment:
Please read about the use-case for `alt`, we need a more user friendly text here